### PR TITLE
feat: apply job list api polling to CollectorDetailPage

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

https://github.com/cloudforet-io/console/assets/26986739/768bd6b2-6120-401e-a9d7-65f3507c18c1

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue [hunk 1]
```diff
@@ -214,6 +217,23 @@ const handleDeleteModalConfirm = async () => {
 const handleUpdateEditModalVisible = (value: boolean) => {
     state.editModalVisible = value;
 };
+
+/* Api polling */
+const fetchRecentJob = async () => {
+    if (!collectorJobState.collector) return;
+    await collectorJobStore.getRecentJob();
+};
+const { pause, resume } = useTimeoutPoll(fetchRecentJob, 5000);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue:226]: praise: 😮 useTimeoutPoll!!!

**Review by yuda110 — APPROVED** [review]
LGTM!! nice usage of vueuse

File: apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue (hunk 1 above)
Comment (skdud4659) [reply apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue:226]: 😯

Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue:226]: praise: wow.. 😦

**Review by piggggggggy — COMMENTED** [review]
LGTM!

**Review by piggggggggy — APPROVED** [review]

File: apps/web/src/services/asset-inventory/collector/collector-detail/collector-job-store.ts [hunk 2]
```diff
@@ -12,8 +12,7 @@ const jobQueryHelper = new ApiQueryHelper().setPageLimit(1).setSort('created_at'
 export const useCollectorJobStore = defineStore('collector-job', {
     state: () => ({
         collector: null as null|CollectorModel,
-        recentJob: null as null|JobModel,
-        loading: true,
+        recentJob: undefined as undefined|null|JobModel, // if undefined, it means that the first request is not yet finished
```
Comment (skdud4659) [inline apps/web/src/services/asset-inventory/collector/collector-detail/collector-job-store.ts:15]: 👍🏻

**Review by skdud4659 — APPROVED** [review]
LGTM~!

**Pull request merged**